Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -18,9 +18,11 @@ private struct LoginListUX {
 }
 
 private extension UITableView {
-    var allIndexPaths: [IndexPath] {
-        return (0..<self.numberOfSections).flatMap { sectionNum in
-            (0..<self.numberOfRows(inSection: sectionNum)).map { IndexPath(row: $0, section: sectionNum) }
+    var allLoginIndexPaths: [IndexPath] {
+        return ((LoginsSettingsSection + 1)..<self.numberOfSections).flatMap { sectionNum in
```

Comment:
The crash was due to trying to delete section number 0, rather than starting from section 1